src/java/net/java/sen/dictionary/Morpheme.java

```diff
@@ -348,7 +348,12 @@ public String toString() {
     partOfSpeechData.append((pronunciations.size() > 0) ? pronunciations.get(0) : "null");
     return partOfSpeechData.toString();
   }
-  
+
+  @Override
+  public int hashCode() {
```

Comment:
It is not required by Lucene 6.2.1. I just wanted to suppress the warning message like the following:
```
[javac] lucene-gosen/src/java/net/java/sen/dictionary/Reading.java:28: warning: [overrides] Class Reading overrides equals, but neither it nor any superclass overrides hashCode method
```
I don't think this method should be included in this change, so we can eliminate this addition.